ci/test.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+echoinfo() { echo "[$(date -Is)] - $@" 1>&2; }
+
+echoinfo "Starting test"
+# These directories are used by other steps so make sure the non-root user has access
+chown -R ubuntu:ubuntu /workspace/
+chown -R ubuntu:ubuntu /cache/
+
+# Need to switch users since we can't pass the right flag to allow running Cassandra as root
+sudo -i -u ubuntu bash << EOF
+echoinfo() { echo "[\$(date -Is)] - \$@" 1>&2; }
+export JAVA_HOME="/usr/lib/jvm/java-1.8.0-openjdk-amd64"
+export PATH=$PATH:$JAVA_HOME/bin
+export MAVEN_OPTS="-Dmaven.repo.local=/cache/.m2"
+
+cd /workspace
+
+# create temp directories to avoid issues with concurrent execution
+cp -R . /tmp/$PERSISTENCE_BACKEND
+cd /tmp/$PERSISTENCE_BACKEND
+echoinfo "Copied directory"
+
+PROFILE=""
+case "$PERSISTENCE_BACKEND" in
+  "cassandra-3.11") PROFILE="it-cassandra-3.11,!it-cassandra-4.0"  ;;
+  "cassandra-4.0")  PROFILE="it-cassandra-4.0,!it-cassandra-3.11"  ;;
+  "dse-6.8")        PROFILE="dse,it-dse-6.8,!it-cassandra-4.0,!it-cassandra-3.11" ;;
+esac
+
+echoinfo "Using backend $PERSISTENCE_BACKEND"
+
+export CCM_CLUSTER_START_TIMEOUT_OVERRIDE=600
+mvn -B failsafe:integration-test --file pom.xml -P "${PROFILE}" \
```

Comment:
I suspect running goal like `failsafe:integration-test` ignores profile selectors. For example, in the latest build we still have:
```
Step #6 - "dse-6.8": INFO  [main] 2020-10-15 19:24:58,105 ExternalStorage.java:165 - About to run /JUnit Jupiter/CassandraPersistenceIT/testUpdateWithTTL() with storage cluster version 3.11
```

I suspect we may have to run the full `verify` lifecycle for the profiles to be applied correctly.. hopefully there is a way to ignore just the unit tests in this case.

Reply:
What's worse is that integrations tests for "dse" use C* 3.11 :(

```
Step #6 - "dse-6.8": INFO  [main] 2020-10-15 19:52:12,146 ExternalStorage.java:165 - About to run /JUnit Jupiter/AuthApiTest/authTokenGenerate() with storage cluster version 3.11
```